lib/jekyll-paginate-v2/generator/paginationModel.rb

```diff
@@ -246,6 +246,21 @@ def paginate(template, config, site_title, all_posts, all_tags, all_categories,
           if config['sort_reverse']
             using_posts.reverse!
           end
+
+          # Do this here so the sticky posts still show up in the preferred order, only at the top
+          if config['sticky_first']
+            sticky_posts = []
+            using_posts.delete_if do |post|
+              if post.data['sticky']
+                if @debug
+                  puts "Sticky: ".rjust(20) + post.data['title']
+                end
```

Comment:
You could replace the `if @debug puts..` conditional with:
```ruby
Jekyll.logger.debug "Sticky:", post.data["title"]
```

Reply:
Hmmm. I'm just following the pattern used everywhere else in the code...

Reply:
I see.. then don't change it..

Reply:
*blush*...actually I should probably change the whole debug fiasco I have in there in the first place ;)